corehq/apps/app_manager/templates/app_manager/partials/releases/build_profiles.html

```diff
@@ -22,14 +23,12 @@ <h4 class="panel-title-nolink">
         and options on different phones. Application profiles will appear in <em>new</em> versions
         of the application.
       {% endblocktrans %}
-      <span class="hq-help-template"
-            data-title="{% trans "Application Profiles" %}"
-            data-placement="left"
-            data-content="
-              {% blocktrans %}
-                  Learn more about Application Profiles on our <a href='https://confluence.dimagi.com/display/commcarepublic/Application+Profiles'>Help Site</a>
-              {% endblocktrans %}">
-        </span>
+      {% if can_edit_apps and request|toggle_enabled:"RELEASE_BUILDS_PER_PROFILE" %}
+        <a href="{% url 'manage_releases_by_app_profile' domain %}">{% trans "Manage application profiles here."%}</a>
```

Comment:
nit: `Manage Release for profiles here`? Also should just `here` be the link and not all text?

Reply:
I guess it can work in both ways? I tried to follow what Jenny suggested in the previous comment.

> Application profiles let you install the same application with different languages and options on different phones. Application profiles will appear in versions of the application. **Manage application profiles here** . Read more on our **Help Site**.

Reply:
FWIW I think both options are fine.

Reply:
I am fine with either, was just a suggestion about the link.